amplify/backend/function/fetchMotionState/src/utils.js

```diff
@@ -33,9 +34,23 @@ const getVotingClient = async (colonyAddress) => {
   return colonyClient.getExtensionClient(Extension.VotingReputation);
 };
 
-const getLatestMotionState = async (colonyAddress, motionId) => {
-  const votingReputationClient = await getVotingClient(colonyAddress);
-  return votingReputationClient.getMotionState(motionId);
+const getLatestMotionState = async (
+  colonyAddress,
+  { nativeMotionId, createdBy },
+) => {
+  let motionState = MotionState.Null;
+  try {
+    const votingReputationClient = await getVotingClient(colonyAddress);
+    const isDeprecated = await votingReputationClient.getDeprecated();
+    /* Only fetch state for a motion that was created by the current (active) installation of the voting rep extn */
+    if (!isDeprecated && votingReputationClient.address === createdBy) {
```

Comment:
This ensures we don't get clashes between two motions with the same native id but which were created by different voting rep extns. 

Also ensures the widget gets disabled in the event the voting rep extn is deprecated (in addition to uninstalled).